BEIMA.Backend.Test/DeviceTypeFunctions/GetDeviceTypeTest.cs

```diff
@@ -92,6 +93,9 @@ public void IdIsValid_GetDevice_ReturnsDevice()
             mockDb.Setup(mock => mock.GetDeviceType(It.Is<ObjectId>(oid => oid == new ObjectId(testId))))
                   .Returns(dbDeviceType.GetBsonDocument())
                   .Verifiable();
+            mockDb.Setup(mock => mock.GetFilteredDevices(It.IsAny<FilterDefinition<BsonDocument>>()))
```

Comment:
Let's add a mock verify for this

Reply:
It's very difficult to verify the FilterDefinition, plus this is how we're already doing it in all the other unit tests. What should we verify, that it is not null?

Reply:
Assert.DoesNotThrow(mockDb.Verify(mock => mock.GetFilteredDevices(It.IsAny<FilterDefinition<BsonDocument>>()), Times.Once);

Reply:
I see, I thought you meant to verify the actual filter itself like on line 93. I will add this.

Reply:
fixed f3155ad